providers/src/airflow/providers/edge/provider.yaml

```diff
@@ -14,8 +14,6 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
-
-# yaml-language-server: $schema=../../provider.yaml.schema.json
```

Comment:
Ups, good catch... VSCode user revealed :-D - seems the path is from the structure before providers were extracted from airflow folder. Thanks for the cleanup!